exe/node/Main.hs

```diff
@@ -41,7 +41,7 @@ data Args = Args
     { host          :: HostName
     , gossipPort    :: PortNumber
     , apiPort       :: PortNumber
-    , seeds         :: FilePath
+    , seeds         :: [P2P.NodeAddr Maybe Crypto]
```

Comment:
Small nit (which boils down more to personal taste): I find hard to reason about this type due to the presence of `Maybe`. Once I look at the type it's easy to understand that the first type param is a functor `f`, but it might not be totally obvious for engineers working on code which touches only the boundary of the P2P layer.

I know it's always a delicate balance between indirection & overhead, but do you think we should type alias this guy in a way that the function is concealed, if we expect in non-library code to always be the same? 🤔

Reply:
If you could ask your fresh names oracle for a good name for that type alias? Mine ran out of names and yields only random character sequences...

Reply:
I was leaning towards `NodeAddress`, but that seems error prone if we also don't rename `NodeAddr` 🤔 . `NodeSeed` maybe? It could fit quite well with the current data record field named `seeds`?

Reply:
6a388e3c0fe859704eb5c7ccfb90a6d0d3c876f1